shared/actions/fs/index.js

```diff
@@ -135,6 +135,20 @@ const setTlfSyncConfig = (state, action) =>
     })
   )
 
+const loadSettings = (state, action) =>
+  RPCTypes.SimpleFSSimpleFSSettingsRpcPromise()
```

Comment:
Should we catch the error here so the user doesn't mistake no settings DB for the settings are loading?

I'm not really sure about the conventions around catching RPC errors though, so I may be totally wrong.